AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/SQLStatement+Select.swift

```diff
@@ -19,9 +19,9 @@ struct SelectStatementMetadata {
     let columnMapping: ColumnMapping
     let bindings: [Binding?]
 
-    // TODO remove additionalStatements once sorting support is added to DataStore
     static func metadata(from modelType: Model.Type,
                          predicate: QueryPredicate? = nil,
+                         sort: QuerySortInput? = nil,
                          paginationInput: QueryPaginationInput? = nil,
                          additionalStatements: String? = nil) -> SelectStatementMetadata {
```

Comment:
we cannot remove the `TODO` and not address it. Now that sort is supported we need to replace `additionalStatements` with proper sorting.

If you remove the `additionalStatements: String? = nil` you will find places where a hardcoded "order by" statement is passed. Replace that with a proper `QuerySortInput`